packages/client-core/src/user/VideoWindows/hook.tsx

```diff
@@ -109,15 +109,17 @@ export const useUserMediaWindowHook = ({ peerID, type }: Props) => {
 
   useEffect(() => {
     function onUserInteraction() {
-      videoElement?.play()
-      audioElement?.play()
+      const currentVideoElement = document.getElementById(`${peerID}_video`) as HTMLVideoElement
+      const currentAudioElement = document.getElementById(`${peerID}_audio`) as HTMLAudioElement
+      if (currentVideoElement?.srcObject) currentVideoElement.play()
+      if (currentAudioElement?.srcObject) currentAudioElement.play()
       harkListener?.value?.resume()
     }
     window.addEventListener('pointerup', onUserInteraction)
     return () => {
       window.removeEventListener('pointerup', onUserInteraction)
     }
-  }, [videoElement, audioElement, harkListener?.value])
+  }, [peerID]) // Use peerID to gracefully handle players leaving
```

Comment:
this change doesn't make any sense

Reply:
Dependency array change reverted, the errors from play() are now handled by checking that the video/audio srcObject exists.